<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>701</width>
      <height>403</height>
     </rect>
    </property>
    <widget class="QProgressBar" name="proBar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="filterBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="userBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>User</string>
      </property>
      <widget class="QLineEdit" name="lineEdit1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="userButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Begin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;End&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="locationBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Location</string>
      </property>
      <widget class="QLineEdit" name="lineEdit3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="locationButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Begin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;End&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="timeBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>171</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Time</string>
      </property>
      <widget class="QPushButton" name="timeButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Example:20100607060104</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label9_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Begin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label10">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;End&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="latitudeBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Latitude</string>
      </property>
      <widget class="QLineEdit" name="lineEdit7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="latitudeButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;End&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Begin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <zorder>lineEdit7</zorder>
      <zorder>lineEdit8</zorder>
      <zorder>latitudeButton</zorder>
      <zorder>label5</zorder>
      <zorder>label6</zorder>
     </widget>
     <widget class="QGroupBox" name="longitudeBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>190</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Longitude</string>
      </property>
      <widget class="QLineEdit" name="lineEdit9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="longitudeButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Begin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;End&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>701</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>21</width>
        <height>411</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>21</width>
        <height>411</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label9">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Click the button to begin filter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filterButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="upload"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFilter"/>
   <addaction name="separator"/>
   <addaction name="actionPlot"/>
   <addaction name="separator"/>
   <addaction name="actionCompare"/>
  </widget>
  <action name="upload">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/86159/Desktop/1.jpg</normaloff>C:/Users/86159/Desktop/1.jpg</iconset>
   </property>
   <property name="text">
    <string>filter</string>
   </property>
   <property name="toolTip">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/86159/Desktop/2.jpg</normaloff>C:/Users/86159/Desktop/2.jpg</iconset>
   </property>
   <property name="text">
    <string>plot</string>
   </property>
  </action>
  <action name="actionCompare">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/86159/Desktop/3.jpg</normaloff>C:/Users/86159/Desktop/3.jpg</iconset>
   </property>
   <property name="text">
    <string>compare</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
